MAUI/Circular-Charts/DataLabels.md

```diff
@@ -222,4 +222,63 @@ this.Content = chart;
 
 {% endtabs %}
 
-![Label template in MAUI Chart](DataLabel_images/maui_chart_datalabel_template.png)
\ No newline at end of file
+![Label template in MAUI Chart](DataLabel_images/maui_chart_datalabel_template.png)
+
+## Smart Labels
+
+This feature is used to arrange the data labels smartly and avoid the intersection when there is overlapping of labels. The [SmartLabelAlignment]() enum contains the following values.
+
+* [Shift]() - smartly arranges the overlapped data labels.
+* [Hide]() - hides the intersected data labels.
+* [None]() - intersected data labels will be visible.
+
+By default, value for [SmartLabelAlignment]() property [Shift]().
+
+If the [LabelPlacement](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartDataLabelSettings.html?tabs=tabid-5%2Ctabid-3%2Ctabid-1#Syncfusion_Maui_Charts_ChartDataLabelSettings_LabelPlacement) is [Inner](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DataLabelPlacement.html#Syncfusion_Maui_Charts_DataLabelPlacement_Inner) and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels will shift to outside the slices and arrange smartly. If the [LabelPlacement](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartDataLabelSettings.html#Syncfusion_Maui_Charts_ChartDataLabelSettings_LabelPlacement) is [Inner](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DataLabelPlacement.html#Syncfusion_Maui_Charts_DataLabelPlacement_Inner) and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
+
+If the [LabelPlacement](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartDataLabelSettings.html#Syncfusion_Maui_Charts_ChartDataLabelSettings_LabelPlacement) is [Outer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DataLabelPlacement.html#Syncfusion_Maui_Charts_DataLabelPlacement_Outer) and the [SmartLabelAlignment]() is [Shift](), then the overlapped labels arrange smartly. If the [LabelPlacement]() is [Outer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.DataLabelPlacement.html#Syncfusion_Maui_Charts_DataLabelPlacement_Outer) and the [SmartLabelAlignment]() is [Hide](), then the overlapped labels will be hidden.
+
+If the [SmartLabelAlignment]() is [None](), then the overlapped labels will be visible irrespective of LabelPlacement.
+
+When the [SmartLabelAlignment]() is [Shift](), and if the data label goes out of the chart area, then the labels got trimmed and the tooltip is shown when clicking/tapping the data label. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCircularChart>
+
+    <chart:PieSeries ItemsSource="{Binding Data}" 
+                     ShowDataLabels="True"
+                     XBindingPath="Product" 
+                     YBindingPath="SalesRate">
+         <chart:PieSeries.DataLabelSettings>
+             <chart:CircularDataLabelSettings LabelPlacement="Outer" SmartLabelAlignment="Shift"/>
```

Comment:
@EzhilarasanElangovan31 Similarly, change the API name and its Enum value

Reply:
changed the API name and its Enum value